src/lib/helpers.js

```diff
@@ -12,3 +14,7 @@ export const getAccountLogin = account => {
     return account.auth.login || account.auth.identifier || account.auth.email
   }
 }
+
+export const redirectToStore = () => {
+  cozy.client.intents.redirect('io.cozy.konnectors', { type: 'konnector' })
```

Comment:
Also, using a helper for only this instruction seems a little bit expensive, I am not offended by a Component accessing directly to interapp API. What do you think ?